sccp/imports/scRNA.py

```diff
@@ -154,39 +154,20 @@ def mu_sigma_normalize(geneDF, scalingfactor=1000):
     return normDF, np.log10(means + 1e-10), np.log10(cv + 1e-10)
 
 
-def gene_filter(geneDF, mean, std, offset_value=1.0):
-    """Filters genes whos variance are higher than woudl be predicted by a Poisson distribution"""
-    slope, intercept, _, _, _ = linregress(mean, std)
-    inter = intercept + np.log10(offset_value)
-
-    above_idx = np.where(std > mean * slope + inter)
-    finalDF = geneDF.iloc[
-        :, np.append(np.asarray(above_idx).flatten(), geneDF.shape[1] - 1)
-    ]
-
-    return finalDF, above_idx
-
-
 def gene_import(offset=1.0):
     """Imports gene data from PopAlign and perfroms gene filtering process"""
     genesDF, _ = import_thompson_drug()
-    filteredGeneDF, logmean, logstd = mu_sigma_normalize(genesDF, scalingfactor=1000)
-    if offset != 1.0:
```

Comment:
@andrewram4287 I thought you'd said there were two thresholds, and that you had needed to lower the threshold to capture a few more genes. However, here, it looks like you are removing the filter altogether?

Reply:
So the lowest threshold is 1.0 which keeps all the genes. If we increase the value, genes are removed. However, some of the genes used for cell type assignments are removed when we increase the offset. I can keep the filtering process in case we would like to have it.